homeassistant/components/lifx/services.yaml

```diff
@@ -1,77 +1,168 @@
 set_state:
-  description: Set a color/brightness and possibliy turn the light on/off.
+  name: Set State
+  description: Set a color/brightness and possibility turn the light on/off.
```

Comment:
Typo on `possibly` both in your code, and the original.

Reply:
One spelling error for one grammar error. lol